saleor/core/templatetags/shop.py

```diff
@@ -27,3 +28,38 @@ def get_sort_by_url(context, field, descending=False):
     else:
         request_get['sort_by'] = field
     return '%s?%s' % (request.path, urlencode(request_get))
+
+
+@register.assignment_tag(takes_context=True)
+def get_sort_by_toggle(context, field):
+    '''
+    This templatetag returns data needed for sorting querysets by links.
+    '''
+    request = context['request']
+    request_get = request.GET.copy()
+    sort_by = request_get.get('sort_by')
+    arrow_src = ''
```

Comment:
I would rename it to `sorting_icon`. If we changed from arrows to something else we won't have to update the variable names :)